### ISSUE TITLE
White text on light green cards hard to read

### ISSUE THREAD

**Issue by davekempe** [meerkat-dashboard/meerkat#87]
White text on light green is hard to ready at a distance. Can we have the option to change the font colour added?

**Comment by ollytom** [comment]
I think there's a bit of a [XY problem](https://en.wikipedia.org/wiki/XY_problem) going on here. The
issue is that white text on light green is hard to read i.e. low contrast between text and background. 
One way to raise contrast is to change the green to something darker. Another is to add a text outline (see
http://2.bp.blogspot.com/-TGEt81O_qaI/Uxns70ZKVEI/AAAAAAAACE4/V5v81OF9sT4/s1600/white-text-black-outline.png)

I'm always thinking about ways to solve problems without code haha ;)

**Comment by davekempe** [comment]
Outline is a good idea. We can try with no code, but the font will need to be thick enough. Let me experiment with my mad CSS skills and get back to you.

**Comment by davekempe** [comment]
![image](https://user-images.githubusercontent.com/140519/184517762-b40e62db-5293-4c3b-9b07-96d1622f12c5.png)

![image](https://user-images.githubusercontent.com/140519/184517736-47edcd22-f3b3-4816-8f1a-3976c6ce5b8a.png)
```
.check > .card {
    padding: 6px 10px;
    color: floralwhite;
    outline: 2px dashed blue;
    background-color: var(--color-black);
    display: flex;
    justify-content: center;
    align-items: center;
    text-transform: uppercase;
    white-space: pre-wrap;
    text-shadow: -1px -1px 0 #000, 1px -1px 0 #000, -1px 1px 0 #000, 1px 1px 0 #000;
```
The text-shadow all around gives it this effect. I guess it looks ok on a large single card, but gets a bit messy on smaller cards or text.

I think making the green darker is the simpler solution. Users can already customise the colour of the green, so perhaps we don't need to worry. But allowing an outline in the options might be cool too for some applications. In particular the Static Test element perhaps?

**Comment by davekempe** [comment]
Red is OK I guess

![image](https://user-images.githubusercontent.com/140519/184517824-99c6516b-86eb-483f-bd1a-a10ddaba421d.png)

**Comment by ollytom** [comment]
> I guess it looks ok on a large single card, but gets a bit messy on smaller cards or text.

This may be solved by using `em` instead of `px`. `em` is a relative
measurement to the size of the font, so the result is that the text
outline width scales as the font size changes.

Here's an example using the non-standard `-webkit-text-stroke`
properties. Despite it being non-standard this property is supported
in Firefox 100 on my OpenBSD machine.

	.text {
		text-transform: none !important;
		word-wrap: break-word;
		-webkit-text-stroke-width: 0.03em;
		-webkit-text-stroke-color: black;
	}

I guess there's some sweet spot value for `webkit-text-stroke-width`.
I did some quick fiddling and got to 0.03, but my display on this
thinkpad really sucks.

I set the properties in `.text` but there's almost certainly a more
specific place to set it which would be clearer!
